src/deck/guides/konnect.md

```diff
@@ -107,6 +107,35 @@ deck ping
 
 Successfully Konnected as MyName (Konnect Org)!
 ```
+{% if_version gte:1.14.x %}
+### Authenticate using a personal access token
+
+You can generate a personal access token (PAT) in {{site.konnect_short_name}} to use to authenticate with decK commands. This can be useful for organizations with CI pipelines that can't use the standard username and password authentication. 
+
+Before you generate a PAT, keep the following in mind:
+
+* A PAT is granted all of the permissions that the user has access to via their most up-to-date role assignment.
+* The PAT has a maximum duration of up to 12 months.
+* There is a limit of 10 personal access tokens per user.
+* Unused tokens are deleted/revoked after 12 months of inactivity.
+
+To generate a PAT in {{site.konnect_short_name}}, go to your name > **Personal access tokens** and click **+ Generate token**. After configuring the name and expiration date, make sure you copy the token to a secure location. 
```

Comment:
```suggestion
To generate a PAT in {{site.konnect_short_name}}, select your name to open the context menu 
 and click **Personal access tokens**, then click **Generate token**. After configuring the name and expiration date, make sure you copy the token to a secure location. 
```

I feel like 'go to your name' isn't very clear. but I can't think of anything better so if this idea isn't good let's just leave it.